template/src/components/Login/Login.tsx

```diff
@@ -49,13 +47,15 @@ export const Login: FC<ComponentProps> = ({
         Welcome, please {'\n'}sign in.
       </Text>
       <TextInput
-        placeholder="✉️ User name"
+        placeholder="User name"
```

Comment:
Minor detail:
I see in the registration page below we ask for email. Should we change this to "Email" instead? 🤔

If you disagree, can we change it to "Username"? Sorry, ocd 🙈

Reply:
I did this based on the design but I had the same thought, I will change this to be consistent and then it will open up discussion when we do the work for Hoag

Reply:
Sounds good!